tests/acceptance/helpers.py

```diff
@@ -163,22 +136,39 @@ def create_and_validate_request_and_response(endpoint, method, session, bypass_v
     :return:
         - response: API response object
     """
-    request, request_result = create_and_validate_request(
-        endpoint, method, payload, params, dict(session.headers)
+    base_url = os.getenv('host')
+    headers = dict(session.headers)
+    headers["Content-Type"] = "application/json"
+
+    request = Request(
+        url=base_url + endpoint,
+        method=method,
+        params=params,
+        data=payload,
+        headers=headers,
     )
 
-    if not bypass_validation_request:
-        request_result.raise_for_errors()
 
-    base_url = os.getenv('host')
+  
+    try:
+        create_and_validate_request(request)
+    except Exception as e:
```

Comment:
Could we try to narrow down which exceptions might occur and use those? instead of a generic Exception.

Reply:
Thanks @Mat001 for the suggestions
updated with specific exceptions now